src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -64,15 +69,28 @@ export function QueryPageContainerComponent(): JSX.Element {
                 uniqueValuesObject[id] = filterElement.values;
             });
 
+            const nrOfSelectedIsolates = isolateCountProp.totalNumberOfIsolates;
+
             setData({
+                ...data,
                 ZNData: adaptedDbIsolates,
                 ZNDataFiltered: adaptedDbIsolates,
                 keyValues: DbKeyCollection,
                 uniqueValues: uniqueValuesObject,
+                nrOfSelectedIsolates,
             });
         }
     };
 
+    const fetchIsolateCounted = async (): Promise<void> => {

```

Comment:
duplicated code. You could use this function above

Reply:
In the final version this function is no longer in the queryPage container. So this comment is obsolete.